database/seeder.js

```diff
@@ -14,6 +14,20 @@ const sequentialDate = (next) => {
   return `${year}-${month}-${day}`;
 };
 
+// in the future I will set each hotel as a new instance of the mongoose model Hotel
+// let hotelReservation = new Hotel({id : hotels}) this will set
+// the id as the current hotelId i am working with
+//
+// I will set each room as a new instance of the mongoose model Booking
+// const reservation = new Booking(currentRoom)
+// whereupon I will push them into the hotel using hotelReservation.rooms.push(reservation)
+// as I decrement hotels I will then save the hotel model using
+// hotelReservation.save()
+// ^ this will not need to call the db method anymore (will import mongoose instead)
+// since hotel is being defined as a model in this scope.
+// alternatively i can call db.save and change it to
+// save (reservation)
+// reservation.save() and save it directly
```

Comment:
good job with making a plan for yourself =). Don't forget to take these comments out when you are done implementing them =)